README.md

```diff
@@ -72,18 +72,13 @@ By default this will generate 1 file and 1 folder in the current directory:
 
 ### Running the node
 
-To run the node you will need to provide a configuration file. We have an example config file in [node/miden-node.toml](bin/node/miden-node.toml). Then, to run the node, run:
+Create a configuration file based on [node/miden-node.toml](bin/node/miden-node.toml), then create the necessary directories and start the node:
 
 ```sh
+mkdir -p /opt/miden
 miden-node start --config <path-to-config-file> <component-to-be-started>
 ```
```

Comment:
I think we may need to update the docs more comprehensively here as the previous steps assume that everything is in the current directory (rather than in `/opt/miden`). In general, I think we have the following ways of installing the node:

1. By cloning the repo and compiling the source.
2. By running `cargo install`.
3. By building and running docker container (I wonder if these should be two separate steps and we'd provide docker image for download).
4. By downloading and running `.deb` package.

I think only the first 2 (or 3 if we include building of Docker image) require Rust to be installed. And for `cargo install`, I think we should mention explicitly which files need to be copied where.

Reply:
created an issue: https://github.com/0xPolygonMiden/miden-node/issues/355